src/type-generator.ts

```diff
@@ -361,15 +361,17 @@ export class TypeGenerator {
       this.emitDescription(code, fqn, def.description);
 
       code.openBlock(`export class ${typeName}`);
+      const possibleTypes = [];
 
       for (const type of options) {
+        possibleTypes.push(type[0]+type.substr(1));
```

Comment:
Would this have the same effect?
```suggestion
        possibleTypes.push(type);
```

Reply:
whoops, yes. Fixed, thanks!